### DISCUSSION TITLE
Sampling for JS/TS SDKs and integrations

### DISCUSSION THREAD

**Discussion by marcklingen** [langfuse/langfuse#3529] [Ideas]
### Describe the feature or potential improvement

Currently, sampling is only support by Python intgerations. Let's use this thread to track overall interest in sampling for JS/TS.

**Comment by hassiebp** [comment]
This has been shipped: 
- [Release](https://github.com/langfuse/langfuse-js/releases/tag/v3.34.0)
- [Changelog](https://langfuse.com/changelog/2025-01-30-sampling-js-ts-sdk)
